sys_conn_helper_freebsd.go

```diff
@@ -14,6 +14,8 @@ const (
 	ipv4PKTINFO  = 0x7
 )
 
+const ecnIPv4DataLen = 4
```

Comment:
This is wrong: It should be 1 for FreeBSD.  See https://cgit.freebsd.org/src/tree/sys/netinet/udp_usrreq.c#n1144
4 is the correct length for use with `setsockopt(2)` but not as a cmsg.